codalab/rest/legacy.py

```diff
@@ -173,13 +175,13 @@ def get_bundle_file_contents(self, uuid):
         Otherwise, get stdout and stderr.
         For each file, return a truncated version.
         """
-        def get_summary(info, name):
+        def get_summary(download_manager, info, name):
             if info['type'] == 'file':
                 TRUNCATION_TEXT = (
                     '\n'
                     '... Truncated. Click link above to see full file. ...\n'
```

Comment:
It's helpful to have some message telling the user where they can find the non-truncated version.